blackbaud/skyux2
# Demo components module

This branch essentially moves all demo components to an exported Demo module that can be consumed by other SPAs.

- Eventually, we will remove the documentation from this SPA, and move it elsewhere.

**How to test:**

Clone the docs SPA:
- Clone https://github.com/blackbaud/skyux2-docs
- Switch branch to `first-release`

Build UX components:
- Type `npm run build`
- Copy the `dist` folder into `skyux2-docs/node_modules/@blackbaud/skyux`
- Type `skyux build -s` to see the new demo components
